features/resolution.yml.dist

```diff
@@ -0,0 +1,53 @@
+# Generated from: resolution.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
+  support:
+    chrome: "68"
+    chrome_android: "68"
+    edge: "79"
+    firefox: "62"
+    firefox_android: "62"
+compat_features:
+  # baseline: low
+  # baseline_low_date: 2022-09-12
+  # support:
+  #   chrome: "29"
+  #   chrome_android: "29"
+  #   edge: "12"
+  #   firefox: "8"
+  #   firefox_android: "8"
+  #   safari: "16"
+  #   safari_ios: "16"
+  - css.at-rules.media.resolution
```

Comment:
I'm very suspicious of this data. MDN says a unit is required. I think this whole feature is actually busted in Safari, or at very least it hasn't been tested recently. See https://bugs.webkit.org/show_bug.cgi?id=78087

I suspect we'll want to land the rest of this, but split this feature into its own PR while we fix BCD.

Reply:
Moved to https://github.com/web-platform-dx/web-features/pull/1798